spec.md

```diff
@@ -1665,15 +1749,26 @@ module MyModule :
   out[4] <= in
 ```
 
+The following example is the same as above, but with a constant vector.
+
+``` firrtl
+module MyModule :
+  input in: const UInt
+  output out: const UInt[10]
+  out[4] <= in ; out[4] has a type of const UInt
+```
+
+
 ## Sub-accesses
 
 The sub-access expression dynamically refers to a sub-element of a vector-typed
 expression using a calculated index. The index must be an expression with an
-unsigned integer type.  An access to an out-of-bounds element results in an 
-indeterminate value (see [@sec:indeterminate-values]).  Each out-of-bounds 
-element is a different indeterminate value.  Sub-access operations with constant
-index may be convereted to sub-index operations even though it converts
-indeterminate-value-on-out-of-bounds behavior to a compile-time error.
+unsigned integer type.  If the expression is of a constant vector type, the 
+sub-element shall be of a constant type.  An access to an out-of-bounds element 
+results in an indeterminate value (see [@sec:indeterminate-values]).  Each 
```

Comment:
Is this saying that the indeterminate value is `const` ?

Reply:
This is a good question.  I believe it works as one expects.  An out-of-bounds access would be an indeterminate value of const type.  In practice, it's either going to be 0 or one of the in-bound inputs, both of which are const in this scenario.